gb-cpu/src/error.rs

```diff
@@ -0,0 +1,7 @@
+#[derive(Debug)]
+pub enum Error {
+    InvalidPC(u16),
+    IllegalSet(usize, u8),
+    SegmentationFault(u16),
+    Unimplemented,
```

Comment:
What's the goal of this error `Unimplemented` ? you could try to use the macro `unimplemented!()` or `todo!()`

Reply:
The macro `unimplemented!()`/`todo!()` will stop the program, with this variant I think we can still run the emulation even if the implementation of the opcode is incomplete.